DegreeProject/Assets/Scripts/Player/Player.cs

```diff
@@ -4,7 +4,7 @@
 using Mirror;
 
 [RequireComponent(typeof(PlayerController))]
-public class Player : NetworkBehaviour
+public partial class Player : NetworkBehaviour
```

Comment:
Vad är anledningen för att köra med en `partial` klass? 
Många gånger när du känner att du behöver dela upp en klass i olika filer, så är det ett tecken på att klassen i sig har hand om för många områden. En klass så som `Player` tenderar att lätt bli en misc klass som tar hand om allt som inte passar in i de andra Player-relaterade klasserna. 

Se om du kan komma på vad Player klassen är skapad att ha hand om, och försök gå djupare än "den har hand om spelaren".